src/Wpdb/AbstractBaseWpdbSelectResourceModel.php

```diff
@@ -399,6 +393,42 @@ protected function _setMapFactory(MapFactoryInterface $mapFactory)
         $this->mapFactory = $mapFactory;
     }
 
+    /**
+     * Creates the result set from raw record data sets.
+     *
+     * @since [*next-version*]
+     *
+     * @param array|stdClass|Traversable $rawResults The list of raw record data sets.
+     *
+     * @return MapInterface[]|stdClass|Traversable A list of maps, each containing data for a record.
+     */
+    protected function _createResultSet($rawResults)
+    {
+        $results = [];
+
+        foreach ($rawResults as $_rawResult) {
+            $results[] = $this->_createResult($_rawResult);
+        }
+
+        return $results;
+    }
+
+    /**
+     * Creates the result map from the raw data of a record.
+     *
+     * @since [*next-version*]
+     *
+     * @param array|stdClass|ArrayObject|null $rawResult The raw data for the record.
```

Comment:
`null` does not appear to be a possible value.

Reply:
uh oh, looks like reflex